### ISSUE TITLE
Sea Creatures: Mosasaur is still sitting and Liopleurodon doesn't attack if happy

### ISSUE THREAD

**Issue by Nathairius** [FossilsArcheologyRevival/FossilsArcheologyRevival#465]
Okay so I spawned a Mosasaur and I exited and went back into the world and it was sitting on the bottom of the ocean (not sleeping). As of right now it's still sitting. I have not encountered the bug with any other prehistoric mob though. Also the Liopleurodon and Mosasaur is supposed to be aggressive and attack even when unprovoked but the untamed ones don't attack me if they're happy. Aren't they supposed to attack?

![modlist](https://cloud.githubusercontent.com/assets/20911297/17501844/51a193cc-5da8-11e6-93b5-ff626a49921d.png)
![2016-08-08_20 39 32](https://cloud.githubusercontent.com/assets/20911297/17501850/696a6c7c-5da8-11e6-82b1-0185df49bbb4.png)
![2016-08-08_20 39 22](https://cloud.githubusercontent.com/assets/20911297/17501851/696f76ae-5da8-11e6-8e76-e6b0499c59d9.png)
![2016-08-08_20 39 14](https://cloud.githubusercontent.com/assets/20911297/17501852/696fd0a4-5da8-11e6-9da2-5f1ca127737e.png)
![2016-08-08_19 42 53](https://cloud.githubusercontent.com/assets/20911297/17501015/7a045db6-5da0-11e6-97f2-a55eee2300e3.png)
![2016-08-08_13 06 51](https://cloud.githubusercontent.com/assets/20911297/17500387/afaeb55c-5d9a-11e6-887e-e0393a15ebad.png)
![mozazaur](https://cloud.githubusercontent.com/assets/20911297/17501726/5bd1f3f6-5da7-11e6-915e-bd4502d62123.png)
![liopleurodon](https://cloud.githubusercontent.com/assets/20911297/17501727/5cf611e0-5da7-11e6-9f4b-bc69a3ce381d.png)

**Comment by PCAwesomeness** [comment]
I believe you are looking at outdated info.

**Comment by Nathairius** [comment]
Yeah nevermind I guess it was just the one. I spawned more Mosasaurs, exited, came back in, and they were swimming.

**Comment by Nathairius** [comment]
Okay it appears to be certain Mosasaurs that end up sitting when I go back in. It's not all of them but some of them

**Comment by Nathairius** [comment]
It's all of the Mosasaurs now. 

![2016-08-08_22 44 02](https://cloud.githubusercontent.com/assets/20911297/17503927/c3d79692-5db9-11e6-88fb-7c669842a319.png)

**Comment by crakiMDH** [comment]
I can't see anything in the picture that you are stating all of them are sitting in. In fact, I can't see ANYTHING of Mosasaurus in any picture, even with full brightness on. I also have noticed that in the mods list picture, it says Fossils and Archeology 7.3.1. Try switching to 7.3.2. (P.S if the issue still happens, please take the pics with night vision so the devs can also see it)

**Comment by Nathairius** [comment]
I know it says that on the screen but I've updated I swear. Here's a screenshot to prove it:
![2016-08-09_16 43 49](https://cloud.githubusercontent.com/assets/20911297/17534839/dd2bb122-5e50-11e6-82bb-ab6e4abe9cdf.png)

**Comment by Nathairius** [comment]
Okay so I turned up the brightness. It was on "Moody" so I apologize. And apparently when I logged out and back in the first time a few had dropped to the bottom but some were still swimming. But the second time I logged out and came back in they were all at the bottom. 
![2016-08-09_16 50 04](https://cloud.githubusercontent.com/assets/20911297/17535002/c846ba08-5e51-11e6-98e8-01a988d6834c.png)
![2016-08-09_16 50 58](https://cloud.githubusercontent.com/assets/20911297/17535004/c84aae74-5e51-11e6-90c3-3bf2c15d6d27.png)
![2016-08-09_16 50 55](https://cloud.githubusercontent.com/assets/20911297/17535003/c849b488-5e51-11e6-9701-d85efef72519.png)

**Comment by crakiMDH** [comment]
Hmmm... Try spawning in 7.3.2 Mosasaurus. Also, I told you to use night vision so everyone can easily see it.

**Comment by Nathairius** [comment]
But I've updated. I wouldn't be able to spawn Ichthyosaurus otherwise right? And here are some more pictures with night vision:
![2016-08-09_17 02 26](https://cloud.githubusercontent.com/assets/20911297/17535295/6040b5b0-5e53-11e6-92fb-c8b5ad64d823.png)
![2016-08-09_17 02 58](https://cloud.githubusercontent.com/assets/20911297/17535305/62d4050c-5e53-11e6-80d4-8cc290c6be2b.png)
![Uploading 2016-08-09_17.03.21.png…]()

**Comment by crakiMDH** [comment]
But have you tried spawning in 7.3.2 Mosasaurus? Perhaps they are looking for food, check if dino starve config is set to true. Or maybe they are just chilling and sitting down? Was it night time when this happens? Cause some creatures sit down right before going to sleep.

**Comment by Nathairius** [comment]
Okay I have proven I am in 7.3.2 twice already. I'm not certain why it says I'm in 7.3.1 when I go into "Mods" but the file is 7.3.2 and I can spawn Ichthyosaurus along with the other new mobs. Am I maybe missing something? Is there something you have to toggle? And yeah I tested it and they sit even when they aren't sleeping. They move around at first and then by the second time I log out and back in they are all sitting even when not sleeping
